build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
this doesn't use pipes so I assume it's a precaution?

Reply:
Ya this is just my "standard" Bash configuration. Also see https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/ (`-x` is too noisey for me though).